src/EnergyPlus/IOFiles.hh

```diff
@@ -275,8 +275,9 @@ public:
             if (specs_.precision > 1) {
                 // reduce the precision to get rounding behavior
                 --specs_.precision;
-            } else {
+            } else if (specs_.precision != -1) {
                 // We need AT LEAST one in precision so we capture a '.' below
+                // If specs_.precision is -1, we don't need to do anything, keep the fmt defaults which are sane
```

Comment:
In the Z case, if the precision wasn't set already, then let fmt default it.

Reply:
Sounds reasonable, if we want control then we just need to be explicit about the precision we want.

Reply:
Is this the default precision for the Fortran output? This whole code is to emulate Fortran output styles within the fmt library.